yarn.lock

```diff
@@ -10116,6 +10148,15 @@ react-is@^17.0.1:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-17.0.1.tgz#5b3531bd76a645a4c9fb6e693ed36419e3301339"
   integrity sha512-NAnt2iGDXohE5LI7uBnLnqvLQMtzhkiAOLXTmv+qnF9Ky7xAPcX8Up/xWIhxvLVGJvuLiNc4xQLtuqDRzb4fSA==
 
+"react-reconciler@>=0.26.0 <0.27.0", react-reconciler@^0.26.0:
```

Comment:
A bunch of these can technically be deduplicated, I'd probably run`npx yarn-deduplicate` with the `--packages` flag to avoid these new duplicates.